content/Day-7/_index.md

```diff
@@ -0,0 +1,84 @@
+
+**Network-3** defines how internetworks (interconnected networks) function. This network layer is the lowest one in the OSI model that is concerned with actually getting data from one computer to another, even if it is on a remote network. In contrast, the data link layer only deals with devices that are local to each other via the LAN.
+
+**Transport-4** the transport layer acts as a liaison between the abstract world of applications and the layers 1 to 3. layers 1 through 3 are concerned with the actual packaging, addressing, routing, and delivery of data to other machines. 
+
+**Session-5** layer is responsible for addressing, packaging, and delivery of data. This layer is mainly concerned with software application issues and not with the details of network (the functions of layers 1-3)
+
+**Presentation-6** this layer is charged with taking care of any issues that might arise when data sent from one system needs to be viewed (presented) in a different way by the receiving system. For example, Windows OS, MAC OS and Linux systems all have similar functionality but different file systems architecture. So data sent to each system will need to be translated to the specific machine instructions. 
+
+**Application-7** It's important to understand that the OSI model application layer is not the same as an app or application. In the OSI model, this layer provides services for user applications to use. For example, when we use our web browser, that actual software is an application. It doesn't really reside at the application layer. It makes use of the services offered by a protocol that operates at the application layer, which is called the Hypertext Transfer Protocol (HTTP or HTTPS). 
+
+Understanding the OSI model can help us reduce the number of hours we spend troubleshooting down from hours to minutes. Once we know how our application is communicating and what our application is communicating with, we can take the appropriate steps to resolve any issues we find within the application. Knowing where to look is more important that knowing all the steps needed to resolve the problem. After all, the steps are dynamic. 
+
+## Network Terms 
+
+Other network terms jargon you should be familiar with are listed below. This is intended to be a reference and not an exhaustive list. We expect that you will take this list and build upon it. Below are just a few examples of the many protocols that make up the TCP/IP suite. These protocols enable a seamless communication and data exchange that we experience on the internet and local networks.
+
+**TCP/IP**, Transmission Control Protocol/Internet Protocol a suite of networking protocols that form the basis for communication on the internet and private networks. It provides a set of rules and conventions for how data should be sent, received, and routed between devices on a network. 
```

Comment:
Similar to above the following sections can be 3-level headers for clarity / visual